lib/ansible/config/base.yml

```diff
@@ -215,537 +224,635 @@ ANSIBLE_SSH_PIPELINING:
 DEFAULT_BECOME_EXE:
+  name: Choose 'become' executable
   default: ~
   description: 'executable to use for privilege escalation, otherwise Ansible will depend on PATh'
   env: [{name: ANSIBLE_BECOME_EXE}]
   ini:
   - {key: become_exe, section: privilege_escalation}
-  yaml: {key: privilege_escalation.become_exe}
 DEFAULT_BECOME_FLAGS:
+  name: Set 'become' executable options
   default: ~
   description: Flags to pass to the privilege escalation executable.
   env: [{name: ANSIBLE_BECOME_FLAGS}]
   ini:
   - {key: become_flags, section: privilege_escalation}
-  yaml: {key: privilege_escalation.become_flags}
 DEFAULT_BECOME_USER:
-  default: root
-  description: User your become when using privilege escalation, most systems will use 'root' when no user is specified.
+  name: Set the user you 'become' via privlege escalation
+  default: ~
+  description: The user your login/remote user 'becomes' when using privilege escalation, most systems will use 'root' when no user is specified.
   env: [{name: ANSIBLE_BECOME_USER}]
   ini:
   - {key: become_user, section: privilege_escalation}
-  yaml: {key: privilege_escalation.become_user}
+  yaml: {key: become.user}
 DEFAULT_CACHE_PLUGIN_PATH:
+  name: Cache Plugins Path
   default: ~/.ansible/plugins/cache:/usr/share/ansible/plugins/cache
-  description: 'TODO: write it'
+  description: Colon separated paths in which Ansible will search for Cache Plugins.
   env: [{name: ANSIBLE_CACHE_PLUGINS}]
   ini:
   - {key: cache_plugins, section: defaults}
   type: pathlist
-  yaml: {key: defaults.cache_plugins}
 DEFAULT_CALLABLE_WHITELIST:
+  name: Template 'callable' whitelist
   default: []
-  description: 'TODO: write it'
+  description: Whitelist of callable methods to be made available to template evaluation
   env: [{name: ANSIBLE_CALLABLE_WHITELIST}]
   ini:
   - {key: callable_whitelist, section: defaults}
   type: list
-  yaml: {key: defaults.callable_whitelist}
 DEFAULT_CALLBACK_PLUGIN_PATH:
+  name: Callback Plugins Path
   default: ~/.ansible/plugins/callback:/usr/share/ansible/plugins/callback
-  description: 'TODO: write it'
+  description: Colon separated paths in which Ansible will search for Callback Plugins.
   env: [{name: ANSIBLE_CALLBACK_PLUGINS}]
   ini:
   - {key: callback_plugins, section: defaults}
   type: pathlist
-  yaml: {key: defaults.callback_plugins}
+  yaml: {key: plugins.callback.path}
 DEFAULT_CALLBACK_WHITELIST:
+  name: Callback Whitelist
   default: []
-  description: 'TODO: write it'
+  description:
+    - List of whitelisted callbacks, not all callbacks need whitelisting,
+      but many of those shipped with Ansible do as we don't want them activated by default.
   env: [{name: ANSIBLE_CALLBACK_WHITELIST}]
   ini:
   - {key: callback_whitelist, section: defaults}
   type: list
-  yaml: {key: defaults.callback_whitelist}
+  yaml: {key: plugins.callback.whitelist}
 DEFAULT_CONNECTION_PLUGIN_PATH:
+  name: Callback Plugins Path
```

Comment:
Wrong name here, dupes with name for DEFAULT_CALLBACK_PLUGIN_PATH

Reply:
fixed